Userland/Libraries/LibJS/Runtime/TypedArray.cpp

```diff
@@ -516,6 +516,10 @@ void TypedArrayBase::visit_edges(Visitor& visitor)
         auto& vm = this->vm();                                                                                              \
         Base::initialize(realm);                                                                                            \
                                                                                                                             \
+        /* NOTE: This is as these functions should be implemented by the TypedArrayConstructor, and not be defined here */  \
+        storage_delete(vm.names.from);                                                                                      \
+        storage_delete(vm.names.of);                                                                                        \
```

Comment:
Wait why do we need to add the properties in the first place? :thonk:

Reply:
Thanks for the suggestion, that definitely seems like the correct fix here!